packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -526,11 +525,43 @@ export class ProjectService {
         return buildQuery({
             explore,
             compiledMetricQuery,
-            allResults,
             warehouseClient,
         });
     }
 
+    static metricQueryWithLimit(
+        metricQuery: MetricQuery,
+        csvLimit: number | null | undefined,
+    ): MetricQuery {
+        const MAX_CELLS = 100000;
```

Comment:
Should we make this an env var ? so we and self-hosted users can easily change it. Or are we going to have a separate PR for that?

Reply:
I'd say on a separate PR. We don't need it right now